dev-util/pycharm-community/pycharm-community-2024.3.2.ebuild

```
@@ -0,0 +1,231 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop readme.gentoo-r1 wrapper xdg-utils
```

```suggestion
inherit desktop readme.gentoo-r1 wrapper xdg
```
So you can remove `pkg_postinst` and `pkg_postrm`

---

> So you can remove `pkg_postinst` and `pkg_postrm`

Good catch!